Demo/Demo/Info.plist

```diff
@@ -2,6 +2,12 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>UIAppFonts</key>
```

Comment:
Should these be added to the Demo app's info.plist?

Reply:
I tried removing it from the info.plist and it ends up defaulting to SF Pro. Did some research and in order to add new fonts, we need to add the font files and register it. I did, however, remove medium and bold since they're unused

Reply:
Ah I didn't know earlier, but I need to add the font file in Demo app for it to show in the simulator, so I've gone ahead and done that

Reply:
Is this just due to how we pull in the SDK like a direct/carthage integration or will Cocoapods and SPM merchants need to do the same?

Reply:
I found an interesting post related to ([Add Font In Customize Framework Swift 4.2](https://medium.com/@GoalStack/add-font-in-customize-framework-swift-4-2-150a28bd8482)), maybe we can follow this approach